source/common/router/config_impl.cc

```diff
@@ -467,6 +467,22 @@ VirtualHostImpl::VirtualClusterEntry::VirtualClusterEntry(const Json::Object& vi
   priority_ = ConfigUtility::parsePriority(virtual_cluster);
 }
 
+const VirtualHostImpl* RouteMatcher::findWildcardVirtualHost(const std::string& host) const {
+  for (const auto& iter : wildcard_virtual_host_suffixes_) {
+    const uint32_t i = iter.first;
+    const auto& wildcard_map = iter.second;
+    // >= because *.foo.com shouldn't match .foo.com.
```

Comment:
Isn't it easy to include empty string in the wildcard, as it goes well with the common perception of * in regex-es ? Or am I missing something ?